### PR TITLE
Throw an error when number of threads is 0

### PR DESCRIPTION
**Pull request by zxch3n**
Related issue: #20

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -30,8 +30,6 @@ use js_sys::Promise;
 use spmc::{channel, Receiver, Sender};
 use wasm_bindgen::prelude::*;
 use wasm_bindgen::JsValue;
-
-#[cfg(feature = "no-bundler")]
 use js_sys::JsString;
```
Comment (RReverser) [inline src/lib.rs:33]: I don't think removal of this attribute was necessary, because you're still not using `JsString`?

Comment (zxch3n) [reply src/lib.rs]: sorry, fixed

**Comment by RReverser** [comment]
Actually, given that this is purely invoking JS APIs, let's move this check to `startWorkers` in https://github.com/GoogleChromeLabs/wasm-bindgen-rayon/blob/main/src/workerHelpers.js and https://github.com/GoogleChromeLabs/wasm-bindgen-rayon/blob/main/src/workerHelpers.no-bundler.js instead.

That will be a bit cheaper, because it won't require few extra roundtrips to JS, and, correspondingly, will also produce slightly smaller output.

**Comment by zxch3n** [comment]
Done. And make it throw if one uses a negative thread number.

**Comment by zxch3n** [comment]
done

**Comment by RReverser** [comment]
Thanks!

**Pull request merged**